app/views/statuses/_form.html.erb

```diff
@@ -0,0 +1,65 @@
+
+<% if @status.errors.any? %>
+  <div class="alert alert-danger alert-dismissible fade show" role="alert">
+    <h4 class="alert-heading">
+      The following errors prevented the <%= @status.class.name.downcase %> from being saved
+    </h4>
+    <ul>
+      <% @status.errors.full_messages.each do |message|%>
+        <li class="text-start"><%= message %></li>
+      <%end%>
+    </ul>
+    <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+  </div>
+<% end %>
+
+<%= form_with(model: @status, local:true) do |f| %>
+  <div class="card-header fw-bold">
+    <div class="col-4">
+      <div class="form" style="color: black;">
+        <%= f.label :date, class:"form-label text-center", placeholder:"Choose date" %>
+          <%= f.date_field :date, class: "form-control", id: 'date' %>
+      </div>
+    </div>
+  </div>
+  <h3 class="text-center mt-1">Status details</h3>
+    <%= f.fields_for :tasks do |t| %>
+      <%= render 'task_fields' , :f=> t %>
```

Comment:
No need of space before `,` and don't use rocket sign instead write like this `f: t`